datasetinsights/datasets/synthetic.py

```diff
@@ -9,12 +9,13 @@
 
 import datasetinsights.constants as const
 from datasetinsights.data.bbox import BBox2D
-from datasetinsights.data.simulation import AnnotationDefinitions, Captures
-from datasetinsights.data.simulation.download import (
+from datasetinsights.datasets.simulation import (
```

Comment:
Based on the design doc, are we keeping the `simulation` folder? Should this be renamed to `datasetinsights.datasets.unity_perception`?

Reply:
Will go in next PR.